components/Starknet/modules/tools/pages/starkgate_architecture.adoc

```diff
@@ -38,6 +38,17 @@ The _StarkGate Registry_::
 * contains the addresses to all supported bridges
 * enables a bridge developer to stop servicing an existing bridge
 
+[#stark_gate_withdrawal_limit]
+== Withdrawal limit
+
+By default, StarkGate imposes no limit on withdrawals. However, in order tBy default, StarkGate imposes no limit on withdrawals. However, in order to mitigate risks associated with critical vulnerabilities that could result in the loss of user funds, StarkGate can enable a withdrawal limit.
+
+If a serious security issue arises, the security agent in the StarkGate contract can limit withdrawals to 5% of the Total Value Locked (TVL) per day for any affected token by calling the `enableWithdrawalLimit()` function in the StarkGate contract. A dedicated team can then investigate and resolve the issue.
```

Comment:
@natan-granit I need a link to the function in the contract.

Reply:
https://github.com/starknet-io/starkgate-contracts/blob/cairo-1/src/solidity/WithdrawalLimit.sol